src/main/app.js

```diff
@@ -430,6 +431,10 @@ fluid.defaults("gpii.app", {
             funcName: "gpii.app.resetAllToStandard",
             args: ["{that}", "{psp}", "{qssWrapper}.qss"]
         },
+        reApplyPreferences: {
+            funcName: "gpii.app.reApplyPreferences",
```

Comment:
No need for this whole definition and function , you can just bind to the LM's definition directly as follows:
reApplyPreferences: {
    func: "{lifecycleManager}.replayEnvironmentalLogin"
}